### PR TITLE
better error handling for missing model error

### PR DESCRIPTION
**Pull request by TC-42-54**
#### Description of what you did:
Actually, if a model is missing from the configuration but present in the database, you get the following error message: 
`
[2020-10-14T09:17:43.597Z] debug ⛔️ Server wasn't able to start properly.
[2020-10-14T09:17:43.599Z] error TypeError: Cannot read property 'attributes' of undefined
    at Object.getNature (/usr/app/node_modules/strapi-utils/lib/models.js:83:61)
    at Object.keys.forEach.name (/usr/app/node_modules/strapi-connector-bookshelf/lib/mount-models.js:127:21)
    at Array.forEach (<anonymous>)
    at Object.keys.map (/usr/app/node_modules/strapi-connector-bookshelf/lib/mount-models.js:120:40)
`

The problem is it doesn't let you know which model is causing the error.
I just added an error handling statement to show which model is blocking the server from starting.

### CODE REVIEW

File: packages/strapi-utils/lib/models.js [hunk 1]
```diff
@@ -75,6 +75,13 @@ module.exports = {
         });
       });
     } else if (_.has(attribute, 'via') && _.has(attribute, 'collection')) {
+      if (!_.has(models, attribute.collection)) {
+        throw new Error(
+          `The collection \`${_.upperFirst(
+            attribute.collection
+          )}\` is missing from the models ${attribute.plugin ? '(plugin - ' + attribute.plugin + ')' : ''}`
```
Comment (alexandrebodin) [inline packages/strapi-utils/lib/models.js:82]: to make the sentence clearer I would write `the collection x is missing from the plugin x models.`

Comment (TC-42-54) [reply packages/strapi-utils/lib/models.js:82]: Thank you for the feedback, I'm gonna update it !

**Review by alexandrebodin — APPROVED** [review]
LGTM, Thank you for your contribution 💯

**Comment by TC-42-54** [comment]
> LGTM, Thank you for your contribution 💯

You're welcome, it's a pleasure to work with Strapi 😃

**Pull request merged**